resources-slo/strings/strings.xml

```diff
@@ -0,0 +1,58 @@
+<!-- translation credits: Filip Hric -->
+<strings>
+    <string id="Language">slo</string>
+
+    <string id="DateFormatMonthDate">Month-Date eg. Feb 26</string>
+    <string id="DateFormatDayDate">Day-Date eg. Tue 26</string>
+    <string id="MonthDateFormat">$1$ $2$</string>
+    <string id="DayDateFormat">$1$ $2$</string>
+
+    <string id="Hour0">Dvanásť</string>
+    <string id="Hour1">Jedna</string>
+    <string id="Hour2">Dve</string>
+    <string id="Hour3">Tri</string>
+    <string id="Hour4">Štyri</string>
+    <string id="Hour5">Päť</string>
+    <string id="Hour6">Šesť</string>
+    <string id="Hour7">Sedem</string>
+    <string id="Hour8">Osem</string>
+    <string id="Hour9">Deväť</string>
+    <string id="Hour10">Desať</string>
+    <string id="Hour11">Jedenásť</string>
+
+    <string id="TimeMinute0">0 hodín</string>
+    <string id="TimeMinute5">0|aj päť|minút</string>
+    <string id="TimeMinute10">0|aj desať|minút</string>
+    <string id="TimeMinute15">štvrť na|1</string>
+    <string id="TimeMinute20">0 dvadsať</string>
+    <string id="TimeMinute25">0 dvadsať päť</string>
+    <string id="TimeMinute30">0 tridsať</string>
+    <string id="TimeMinute35">0 tridsať päť</string>
+    <string id="TimeMinute40">0 štyridsať</string>
+    <string id="TimeMinute45">tri štvrte na 1</string>
```

Comment:
I try to limiting the lines to 3 as 4 can display poorly
same with: tri štvrte na jednu